collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -1882,17 +1894,6 @@ protected void onPause() {
         Timber.w("onPause %s", Md5.getMd5Hash(getIntent().getData().toString()));
         backgroundLocationViewModel.activityHidden();
 
-        FormController formController = getFormController();
-
-        // make sure we're not already saving to disk. if we are, currentPrompt
-        // is getting constantly updated
-        if (!formSaveViewModel.isSaving()) {
-            if (currentView != null && formController != null
-                    && formController.currentPromptIsQuestion()) {
-                formEntryViewModel.updateAnswersForScreen(getAnswers());
```

Comment:
As far as I can tell, we only need to do this to make sure the save point created in `onSaveInstanceState` will have to update answers. If that's the case, we can move the call from here to `onSaveInstanceState` so it will only happen on recreation rather than when minimizing the app or switching screens.